File: 485-max-consecutive-ones/max-consecutive-ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int max=0;
    for(int i=0,temp=0;i<numsSize;i++){
        if(nums[i]==1){
            temp++;
            if(temp>max) max=temp;
        }else{
            temp=0;
        }
    }
    return max;
}

File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        max = 0
        temp = 0

        for i in range(len(nums)):
            if nums[i] == 1:
                temp += 1

                if temp > max:
                    max = temp

            else:
                temp = 0

        return max           